uint32 id
string type
SidewalkRoutePoint route_point

